articles/virtual-machines/workloads/oracle/configure-oracle-dataguard.md

```diff
@@ -24,10 +24,10 @@ Before you start, make sure that Azure CLI is installed. For more information, s
 
```

Comment:
Re: line 22 ## Prepare the environment
For MS articles H2s are usually followed by a few sentences describing what the customer will learn about or do in this section. Please provide a general description so the customer knows what this section is about.

Reply:
Added the following
"This article demonstrates, step by step, how to deploy Oracle on an Azure virtual machine.  You will then install and configure Data Guard to replicate a database to a secondary (backup) database on a second Oracle instance."